content/exporters/supported-exporters/Python/Honeycomb.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Honeycomb.io (Tracing)"
+date: 2010-03-18
```

Comment:
2019-03-18

Reply:
Just this left @codeboten and then we'll be good to merge.

Reply:
The format that we usually use for Hugo would match `2019-03-18T23:10:00-07:00`, please use that.

Reply:
Updated the format. Will probably want to update the date format on the Go Honeycomb exporter as well: https://github.com/census-instrumentation/opencensus-website/blob/master/content/exporters/supported-exporters/Go/Honeycomb.md

Reply:
Good catch, please feel free to send an update in a separate PR.